src/engine/mode_manager.cpp

```diff
@@ -273,7 +271,8 @@ GameMode *ModeEngine::Get(uint32_t index)
 void ModeEngine::Update()
 {
     // Check whether the fade out is done.
-    if(_fade_out && VideoManager->IsLastFadeTransitional() && !VideoManager->IsFading()) {
+    if(_fade_out && VideoManager->IsLastFadeTransitional() &&
+       !VideoManager->IsFading()) {
```

Comment:
Please use a double indentation to prevent confusion between the end of the condition and the statement.